content/get-started/accessing-data-on-cal-adapt.md

```diff
@@ -16,14 +16,14 @@ There are many ways to access, visualize, and download data from Cal-Adapt:
  
 *Data catalog*: All datasets available on Cal-Adapt are listed in the [data catalog](/data/download/). Search for the dataset you are interested in by entering a search criteria and/or filtering by a climate variable or data producer. You will be prompted to follow a stepwise process to identify a data format (either GeoTIFF or CSV), specify a spatial extent, set aggregation parameters, and then select from different emissions scenarios and climate models. NetCDF files for full spatial extents are available from the [Cal-Adapt file server](https://albers.cnr.berkeley.edu/data/). Links to NetCDF files are displayed on screen during the data format selection step.
  
-*Cal-Adapt file server*: We store climate data provided by California’s scientific and research community in their original formats on the [Cal-Adapt file server](https://albers.cnr.berkeley.edu/data/). You can download NetCDF files of gridded datasets and CSV files of station data from the server. Links to individual datasets are also included in the [data catalog](/data/download/).
+*Cal-Adapt file server*: We store climate data provided by California’s scientific and research community in their original formats on the <a href="https://albers.cnr.berkeley.edu/data/" target="_blank">Cal-Adapt file server</a>. You can download NetCDF files of gridded datasets and CSV files of station data from the server. Links to individual datasets are also included in the [data catalog](/data/download/).
```

Comment:
Replace all instances of `data-catalog` with `Data Download Tool`. I did that in one of my recent PR's so maybe you won't need to.

Reply:
Yeah I was wondering why Nancy was using that language. Ok I'll fix.